ansible/playbook.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: configure frontend servers
+  hosts: _frontend_dev_ujwal_final_project
```

Comment:
where is this hostname coming from?
any idea?

Reply:
i have created two groups one for backend servers and one for frontend server